### PR TITLE
CANS-960 Behavioral/Emotional rating of “1” and “2” appear in summary AR column

### PR DESCRIPTION
**Pull request by nasar-cambria**
https://osi-cwds.atlassian.net/browse/CANS-960

## Description
The summary card domain to include rating 1 and 2 in Action Required column from Behavior/Emotional needs domain.

## Tests

- [x] I used TDD to develop the feature
- [x] I have included unit tests
- [ ] I have included new acceptance tests or modified existing ones
- [ ] I have included other tests

## Types of changes

- [x] New feature (non-breaking change which adds functionality)
- [ ] Refactoring (No behavioral changes)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [x] My code follows the code style of this project.
- [x] I ran all my tests locally which were green.
- [x] My code adds no new issues to Code Climate
- [x] I ran all the linters for the project.
- [x] My code is in a stable state ready to be deployable, but not necessarily complete.
- [x] I promise on my honor as a CWDS developer to ensure I don't break the build (or bring donuts if I leave things broken), to check linters, use best practices, to leave the code in better shape than I found it, and to have a good day.

### CODE REVIEW

**Comment by LeonidMarushevskyi** [comment]
Retest this please

**Comment by vikas199** [comment]
Jenkins retest this please.

**Comment by nasar-cambria** [comment]
Jenkins re test this please

File: app/javascript/Application/components/Assessment/AssessmentSummary/DomainHelper.test.js [hunk 1]
```diff
@@ -64,4 +64,42 @@ describe('DomainHelper', () => {
...
+    })
+
+    it('is false for other domains', () => {
+      expect(isBehavioralNeedsDomain({ code: 'OTHER' })).toBe(false)
+      expect(isBehavioralNeedsDomain({ code: 'STR' })).toBe(false)
+    })
+  })
+
+  describe('itemsValue', () => {
+    it('filters domains and returns values', () => {
```
Comment (wmitchell-cambria) [inline app/javascript/Application/components/Assessment/AssessmentSummary/DomainHelper.test.js]: This test is a good start. Ideally, we should cover more cases, involving multiple domains and filters that actually remove items.

**Pull request merged**